cmake/ValhallaBin2Header.cmake

```diff
@@ -127,7 +127,7 @@ function(BIN2H)
     endif()
 endfunction()
 
-if(CMAKE_SCRIPT_MODE_FILE AND "${CMAKE_SCRIPT_MODE_FILE}" MATCHES "Binary2Header.cmake$")
+if(CMAKE_SCRIPT_MODE_FILE AND "${CMAKE_SCRIPT_MODE_FILE}" MATCHES "ValhallaBin2Header.cmake$")
```

Comment:
whats the significance of the `Valhalla` prefix on all of these?

Reply:
just to make it obvious that its our own modules, not vendored from somewhere like the rest